src/shared/components/tc-communities/communities/iot/JoinSection/index.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * Join button section component for IoT community
+ */
+import React from 'react';
+import PT from 'prop-types';
+import JoinCommunity from 'containers/tc-communities/JoinCommunity';
+import Section from '../../../Section';
+
+import styles from './styles.scss';
+
+const JoinSection = ({ baseUrl }) => (
+  <Section
+    title="Are you ready for Predix?"
+    theme={{
+      container: styles.sectionContainer,
+    }}
+  >
+    <div styleName="buttonContainer">
+      <JoinCommunity competeUrl={`${baseUrl}/challenges?communityId=iot`} label="Join the Topcoder Community for Predix" />
```

Comment:
`competeUrl`? I believe, `JoinCommunity` button does not have such prop.
It only takes theme, label, and communityId, which should be joined. Do I miss something?

Reply:
`competeUrl` is some artifact from the last F2F as there was requirement to show other button linked /challenges for registered community users but this was not implemented good and I removed it.